collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -174,6 +180,51 @@ public static Bitmap getBitmapScaledToDisplay(File file, int screenHeight, int s
         return bitmap;
     }
 
+
+    /**
+     * While copying the file, apply the exif rotation of sourceFile to destinationFile
+     * so that sourceFile with EXIF has same orientation as destinationFile without EXIF
+     */
+    public static void copyImageAndApplyExifRotation(File sourceFile, File destFile) {
```

Comment:
`FileUtils` is growing maybe it would be better to have `ImageFileUtils` for utility methods that operate only on images and then `GeneralFileUtils`? We don't need to add it here but maybe in a separate pr.

Reply:
Agreed. Will create a separate issue followed by a PR after this one.